Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/batteryInput/index.tsx

```diff
@@ -1,4 +1,9 @@
-const BatteryInput = () => {
+interface BatteryInputProps {
+  value: number | string; // Expecting a string for the input value
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void; // Function to handle changes
+}
```

Comment:
You're re-writing this interface in batteryInput and speedInput. Is there a way you can share this interface?